bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecSystemStatus.java

```diff
@@ -62,7 +62,7 @@ public enum SenecSystemStatus {
     NO_SERVER_CONNECTION(38, "NO SERVER CONNECTION"),
     BMS_ERROR(39, "BMS ERROR"),
     MAINTENANCE_FILTER(40, "MAINTENANCE: FILTER"),
-    SLEEPING_MODE(41, "SLEEPING MODE"),
```

Comment:
Don't you need to keep this for compatibility to users not already on the new firmware?

Reply:
The Senechome device only provides the number, openhub displays the name using this enum.